packages/slidev/package.json

```diff
@@ -60,7 +60,6 @@
     "@slidev/client": "workspace:*",
     "@slidev/parser": "workspace:*",
     "@slidev/types": "workspace:*",
-    "@unocss/extractor-mdc": "^0.59.3",
```

Comment:
If this is not in use, it's a mistake. We should add it in the UnoCSS config when mdc is set to true.